01-node-tutorial/answers/09-path-module.js

```diff
@@ -0,0 +1,12 @@
+const path = require('path')
+
+console.log(path.sep)
+
+const filePath = path.join('/content/', 'subfolder', 'test.txt')
+console.log(filePath)
```

Comment:
Beware that leading your first parameter with `/` will make the path from the root of the file system instead of a relative reference from wherever the path is used.

For example, if node was trying to read the file path above it would throw an error (unless you had that path at root). If you exclude the leading forward slash, it would find the file you have relative to this one.

![image](https://user-images.githubusercontent.com/22882623/212208656-8d247e21-92f1-450f-9314-d31d7184e551.png)

Reply:
Ha! I just realized this is how the author of this tutorial wrote the example... not sure why, as it is an unexpected result in my mind! You can ignore me. 😄